app/components/header-bar.js

```diff
@@ -23,17 +24,26 @@ export default Ember.Component.extend({
         return this.get('purecloud').get('me');
     }),
 
-    meJson:computed('purecloud.me', function() {
+    meJson: computed('purecloud.me', function() {
         return JSON.stringify(this.get('purecloud').get('me'),null, "  ");
     }),
 
+    profileImg: computed('purecloud.me', function() {
+      return this.get('purecloud.me') === null || !this.get('purecloud.me.images') ?
+        `${config.APP.urlprefix.replace(/\/?$/, '/')}assets/images/profile-default.svg`
```

Comment:
`urlprefix` will be replaced by the CDN address during building, so this path should work in prod now.